Sources/Sentry/SentryBinaryImageCache.m

```diff
@@ -176,7 +179,11 @@ - (NSInteger)indexOfImage:(uint64_t)address
 - (NSArray<SentryBinaryImageInfo *> *)getAllBinaryImages
 {
     @synchronized(self) {
-        return _cache.copy;
+        if (_cache != nil) {
+            return (NSMutableArray<SentryBinaryImageInfo *> *_Nonnull)_cache.copy;
+        } else {
+            return @[];
```

Comment:
We need to decide if we should just go with no binary images or if we want to indicate an issue by returning `nil`